.env.example

```
@@ -140,6 +140,11 @@ SOURCE_EVIDENCE_PREWARM_INTERVAL_SECONDS=300
 SOURCE_READ_PROJECTION_BACKFILL_ENABLED=true
 SOURCE_READ_PROJECTION_BACKFILL_LIMIT=100
 SOURCE_READ_PROJECTION_BACKFILL_INTERVAL_SECONDS=30
+# Materialize one shared domain health assessment from cached report, DNS, and
+# sender evidence. Browser requests only read this projection.
+HEALTH_SNAPSHOT_REFRESH_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEALTH_SNAPSHOT_REFRESH_ENABLED key should go before the SOURCE_EVIDENCE_PREWARM_ENABLED key

(UnorderedKey)